accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraphConfiguration.java

```diff
@@ -105,6 +119,37 @@ private URI getHdfsRootDir() throws URISyntaxException {
             Map.Entry entrySet = (Map.Entry) entrySetObject;
             configuration.set("" + entrySet.getKey(), "" + entrySet.getValue());
         }
+
+        String hadoopConfDir = getString(HADOOP_CONF_DIR, null);
```

Comment:
can you move all these lines to a method called load hadoop configuration or something.